src/index.js

```diff
@@ -258,6 +283,9 @@ class FlashlsHandler {
       });
 
       this.tech_.on('levelswitch', this.onLevelSwitch_);
+      player.tech_.on('levelswitch', () => {
```

Comment:
use `this.tech_` instead of `player` here and the line below

Reply:
Also this wont get setup if quality levels isnt available. It should be moved to the constructor and not use an anonymous function so that you can also `off` it in the dispose function